packages/react-ai/src/components/AIConversation/context/ActionsContext.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { CustomAction } from '../types';
+
+type ActionsContextProps = CustomAction[] | undefined;
```

Comment:
The base type of a context should not include `undefined`, also prefer to use "props" in interface names for components